### PR TITLE
Update README.md

### LINKED ISSUE #35: For the community

### PR DESCRIPTION
**Pull request by Tyler-Keith-Thompson**
## Checklist:
- [ ] Is the linter reporting 0 errors?
- [ ] Did you comply with our [styleguide](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/STYLEGUIDE.md)?
- [ ] Is there [adequate test coverage](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#test-etiquette) for your new code?
- [ ] Does the CI pipeline pass?
- [ ] Is the [static analysis tool](https://sonarcloud.io/dashboard?id=Tyler-Keith-Thompson_Workflow) reporting 0 issues?
- [ ] Did you [update the documentation](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#documentation)?
- [ ] Did you [update the sample app](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#sample-app)?
- [ ] Do we need to [increment the minor/major version](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#versioning)?
- [ ] Did you [change the public API](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#public-api)?
- [ ] Have you done everything you can to make sure that errors that can occur are compile-time errors, and if they have to be runtime do you have adequate tests and documentation around those runtime errors? [For more details](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#errors).

----

### If Applicable:
- [ ] Did you test when the first item is skipped?
- [ ] Did you test when the last item is skipped?
- [ ] Did you test when middle items are skipped?
- [ ] Did you test when incorrect data is passed forward?
- [ ] Did you test proceeding backwards?

----

### If Public API Has Changed:
- [ ] Did you deprecate (rather than remove) any old methods/variables/etc? [Our philosophy for deprecation](https://github.com/Tyler-Keith-Thompson/Workflow/blob/master/CONTRIBUTING.md#deprecation).
- [ ] Have you done the best that you can to make sure that the compiler guides people to changing to the new API? (Example: the renamed attribute)
- [ ] If necessary, have you tested the upgrade path for at least N-1 versions? For example, if data persists between v1 and v2 then that upgrade should be tested and as easy as we can make it.

### CODE REVIEW

**Comment by Richard-Gist** [comment]
👍

**Review by Richard-Gist — COMMENTED** [review]
I'm commenting here instead of the conversation

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # ActionTesting
 Repository to test out Github actions and building out CI/CD pipelines.
 
-testing milestones
```
Comment (Richard-Gist) [inline README.md:4]: Put this back in please

**Review by Richard-Gist — CHANGES_REQUESTED** [review]
I've added comments and I would like you to fix them

File: README.md (hunk 1 above)
Comment (Richard-Gist) [reply README.md:4]: ok i'll take it anyways

**Review by Richard-Gist — DISMISSED** [review]

**Comment by Tyler-Keith-Thompson** [comment]
@cla-bot I accepted

**Comment by Tyler-Keith-Thompson** [comment]
@cla-bot check

**Review by Richard-Gist — APPROVED** [review]

**Pull request merged**